tests/public_models/test_public_models_predicts.py

```diff
@@ -47,6 +50,33 @@ def test_audio_predict_on_public_models(channel):
         )
 
 
+@both_channels
+def test_text_gen_on_public_llms(channel):
+    """Test text generation for public large language models.
+    All these models can take the same test text prompt.
+    """
+    stub = service_pb2_grpc.V2Stub(channel)
+
+    for title, model_id, app_id, user_id in LLM_TITLE_ID_TUPLES[0]:
+        # test only llama2 model for fast tests execution
```

Comment:
Is this intended to merge? If so, we should just avoid adding the other LLM's in this test as well. It might be best to inference on one of them anyways